deployment/updates/scripts/add_permissions/2015_09_20_update_integration_notify_permission_name.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @package deployment
+ * 
+ * Update integration->notify permission name and related objects
+ */
+ 
+$script = realpath(dirname(__FILE__) . '/../../../../') . '/alpha/scripts/utils/permissions/addPermissionsAndItems.php';
+$config = realpath(dirname(__FILE__)) . '/../../../permissions/partner.0.ini';
+passthru("php $script $config");
+
+$config = realpath(dirname(__FILE__)) . '/../../../permissions/service.integration.integration.ini';
+passthru("php $script $config");
+
+require_once(dirname(__FILE__) . "/../../../../" . "alpha/scripts/bootstrap.php");
+
+$userRole = UserRolePeer::getByNameAndPartnerId(kIntegrationFlowManager::EXTERNAL_INTEGRATION_SERVICES_ROLE_NAME, PartnerPeer::GLOBAL_PARTNER);
+$userRole->setPermissionNames("INTEGRATION_ACTIONS");
```

Comment:
@shpiNi, I would change it to remove `VOICEBASE_ACTIONS` and add `INTEGRATION_ACTIONS` instead of overwriting it.
If this role will be update with a new permission in the future, and this script will be executed, it will override the new permission.
Just explode it by `,`, `trim` the values, unset `VOICEBASE_ACTIONS` and add `INTEGRATION_ACTIONS`